x-pack/legacy/plugins/code/common/lsp_client.ts

```diff
@@ -27,9 +27,7 @@ export class LspRestClient implements LspClient {
     signal?: AbortSignal
   ): Promise<ResponseMessage> {
     try {
-      const response = await kfetch({
-        pathname: `${this.baseUri}/${method}`,
-        method: 'POST',
+      const response = await npStart.core.http.post(`${this.baseUri}/${method}`, {
         body: JSON.stringify(params),
         signal,
```

Comment:
Is  `signal` supported in new platform?

Reply:
yes, it's. https://github.com/elastic/kibana/blob/de5c452f14a5b6ad2fb05baa343889ac36226cdc/src/core/public/http/types.ts#L68

if not we should request them to do so.